packages/map-toolkit/package.json

```diff
@@ -70,6 +72,7 @@
     "vitest-broadcast-channel-mock": "0.1.0"
   },
   "optionalDependencies": {
+    "@accelint/bus": "^1.3.0",
```

Comment:
how do these interact with devDependencies? I'm noticing we're duplicating some but not all. shouldn't client-only also be an optional?